backend/tsconfig.json

```diff
@@ -9,7 +9,7 @@
 		"target": "es2017",
 		"sourceMap": true,
 		"outDir": "./dist",
-		"baseUrl": "../../",
+		"baseUrl": "./",
```

Comment:
what impact would this one have? (just curious)

Reply:
something with vscode (or other editor maybe?) generating correct `import` paths, but so far non of the settings seem to work reliably... so I change this around sometimes and forget to not commit/push this change.. so it ends up in the repo xD